invenio_users_resources/services/groups/tasks.py

```diff
@@ -18,24 +18,26 @@
 
 
 @shared_task(ignore_result=True)
-def reindex_group(role_id):
+def reindex_group(role_ids):
     """Reindex the given user."""
-    index = current_groups_service.record_cls.index
-    if current_groups_service.indexer.exists(index):
-        try:
-            group_agg = GroupAggregate.get_record(role_id)
-            current_groups_service.indexer.index(group_agg)
-        except search.exceptions.ConflictError as e:
-            current_app.logger.warn(f"Could not reindex group {role_id}: {e}")
+    for role_id in role_ids:
+        index = current_groups_service.record_cls.index
+        if current_groups_service.indexer.exists(index):
+            try:
+                group_agg = GroupAggregate.get_record(role_id)
+                current_groups_service.indexer.index(group_agg)
```

Comment:
Moderate: I think that ideally we want to avoid fetching the records one by one and indexing one by one. Right now, the signature of the function changed but the behaviour is the same (a for loop that indexes one by one). 

The indexer has a method called [`bulk_index`](https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/api.py#L254) that allows to index multiple records at once by passing the IDs. Did you already try this option? If not, could you try and let us know if it works?

Reply:
oh I'm stupid
will see give `bulk_index` a go and let you know!